### PR TITLE
[zh-cn] Retranslate CT

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Init translation for certificate transparency.

### Motivation

Previous page was completely wrong and outdated.

### CODE REVIEW

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 1]
```diff
@@ -1,144 +1,40 @@
...
-- `max-age`
-  - : 浏览器应记住仅使用其中一个已定义的密钥访问此站点的时间（以秒为单位）。
-- `includeSubDomains` {{optional_inline}}
-  - : 如果指定了此可选参数，则此规则也适用于所有站点的子域。
-- `report-uri` {{optional_inline}}
-  - : 如果指定了此可选参数，则会将引脚验证失败报告给给定的 URL。
+## 实现
 
-> **备注：** 当前规范要求包含第二个用于备份密钥的引脚，该引脚尚未在生产中使用。这允许更改服务器的公钥，而不会破坏已经记下引脚的客户端的无障碍。例如，当前一个密钥被泄露时，这很重要。
+当证书被提交到 CT 日志时，一个*签名的证书时间戳*（SCT）被生成并返回。这可作为证书已提交的证明，并将被添加到日志中。
```
Comment (yin1999) [inline files/zh-cn/web/security/certificate_transparency/index.md:22]: 我比较倾向于 trustasia 对这个词的翻译：

```suggestion
当证书被提交到 CT 日志时，一个*证书签署时间戳*（SCT）被生成并返回。这可作为证书已提交的证明，并将被添加到日志中。
```

https://www.trustasia.com/view-certificate-transparency/

其它的翻译还有：

- 已签证书时间戳
- 签名证书时间戳

但按照 trustasia 给出的流程图来看，翻译为证书签署时间戳应该更加准确（表达了证书签署的时间）

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 2]
```diff
@@ -1,144 +1,40 @@
...
-> **备注：** 虽然下面的示例显示了如何在服务器证书上设置引脚，但建议将引脚放在颁发服务器证书的 CA 的中间证书上，以简化证书续订和轮换。
+- X.509v3 证书扩展，直接将签名的证书时间戳嵌入叶节点证书中
+- 握手过程中发送的 `signed_certificate_timestamp` 类型的 TLS 扩展
+- OCSP 装订（即 `status_request` TLS 扩展），并提供具有一个或多个 SCT 的 `SignedCertificateTimestampList`
 
-首先，您需要从证书或密钥文件中提取公钥信息，并使用 Base64 对其进行编码。
+通过 X.509 证书扩展，所包含的 SCT 由签发的 CA 决定。自 2021 年 6 月以来，大多数积极使用和有效的公开信任的证书都包含嵌入该扩展的透明度数据。这种方法应该不需要对 web 服务器进行修改。
 
-以下命令将帮助您从密钥文件，证书签名请求或证书中提取 Base64 编码信息。
+使用后一种方法，服务器将需要更新以发送所需的数据。其优点是，服务器运营商可以定制 CT 日志源，提供通过 TLS 扩展/堆叠 OCSP 响应发送的 SCT。
```
Comment (yin1999) [inline files/zh-cn/web/security/certificate_transparency/index.md:32]:
```suggestion
使用后一种方法，服务器将需要更新以发送所需的数据。其优点是，服务器运营商可以定制 CT 日志源，提供通过 TLS 扩展/装订 OCSP 响应发送的 SCT。
```

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 3]
```diff
@@ -1,144 +1,40 @@
...
 
-```plain
-openssl rsa -in my-rsa-key-file.key -outform der -pubout | openssl dgst -sha256 -binary | openssl enc -base64
-```
+## 浏览器需求
```
Comment (yin1999) [inline files/zh-cn/web/security/certificate_transparency/index.md:34]:
```suggestion
## 浏览器要求
```

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 4]
```diff
@@ -1,144 +1,40 @@
...
 
-```plain
-openssl ec -in my-ecc-key-file.key -outform der -pubout | openssl dgst -sha256 -binary | openssl enc -base64
-```
+Google Chrome 要求对 notBefore 日期晚于 2018 年 4 月 30 日签发的证书进行 CT 日志收录。用户将被阻止访问使用不符合规定的 TLS 证书的网站。此前，Chrome 浏览器要求对 _Extended Validation_（EV）和 Symantec 签发的证书进行 CT 收录。
```
Comment (yin1999) [inline files/zh-cn/web/security/certificate_transparency/index.md:36]:
```suggestion
Google Chrome 要求对 notBefore 日期晚于 2018 年 4 月 30 日签发的证书进行 CT 日志收录。用户将被阻止访问使用不符合规定的 TLS 证书的网站。此前，Chrome 浏览器要求对*扩展验证*（EV）和 Symantec 签发的证书进行 CT 收录。
```

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 5]
```diff
@@ -1,144 +1,40 @@
...
 
-```plain
-openssl req -in my-signing-request.csr -pubkey -noout | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | openssl enc -base64
-```
+Apple [需要](https://support.apple.com/en-us/HT205280)多种数目的 SCT，以使 Safari 和其他服务器信任服务器证书。
```
Comment (yin1999) [inline files/zh-cn/web/security/certificate_transparency/index.md:38]:
```suggestion
Apple [要求](https://support.apple.com/en-us/HT205280)多种数目的 SCT，以使 Safari 和其他服务器信任服务器证书。
```

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 6]
```diff
@@ -1,144 +1,40 @@
...
 ---
 
-HTTP 公钥锁定（HPKP）是一种安全功能，它告诉 Web 客户端将特定加密公钥与某个 Web 服务器相关联，以降低使用伪造证书进行 MITM 攻击的风险。
+{{QuickLinksWithSubpages("/zh-CN/docs/Web/Security")}}
 
-为确保 TLS 会话中使用的服务器公钥的真实性，此公钥将包装到 X.509 证书中，该证书通常由证书颁发机构（CA）签名。诸如浏览器之类的 Web 客户端信任许多这些 CA，它们都可以为任意域名创建证书。如果攻击者能够攻击单个 CA，则他们可以对各种 TLS 连接执行 MITM 攻击。HPKP 可以通过告知客户端哪个公钥属于某个 Web 服务器来规避 HTTPS 协议的这种威胁。
+**证书透明度**（Certificate Transparency，CT）是一个开放的框架，旨在监测和防止证书的误发。它在 [RFC 9162](https://www.rfc-editor.org/rfc/rfc9162) 中定义。有了证书透明度机制，新颁发的证书会被“记录”到公开运行的、通常是独立的 *CT 日志*中——这些日志保持着仅添加的、有密码学保证的已颁发 TLS 证书记录。
 
-HPKP 是首次使用信任（TOFU）技术。Web 服务器第一次通过特殊的 HTTP 标头告诉客户端哪些公钥属于它，客户端会在给定的时间段内存储此信息。当客户端再次访问服务器时，它希望证书链中至少有一个证书包含一个公钥，其指纹已通过 HPKP 已知。如果服务器提供未知的公钥，则客户端应向用户发出警告。
+这样一来，证书颁发机构（CA）就可以受到更多的公众审查和监督。潜在的恶意证书，如那些违反 CA/B 论坛*基线要求*的证书，可以更快地被发现和撤销。浏览器供应商和根存储维护者也被授权对他们可能决定不信任的有问题的 CA 做出更明智的决定。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/security/certificate_transparency/index.md:10]:
```suggestion
这样一来，证书颁发机构（CA）就可以受到更多的公众审查和监督。潜在的恶意证书，如那些违反 CA/浏览器论坛*基线要求*的证书，可以更快地被发现和撤销。浏览器供应商和根存储维护者也被授权对它们可能决定不信任的有问题的 CA 做出更明智的决定。
```

File: files/zh-cn/web/security/certificate_transparency/index.md [hunk 7]
```diff
@@ -1,144 +1,40 @@
...
+title: 证书透明度
 slug: Web/Security/Certificate_Transparency
-original_slug: Web/HTTP/Public_Key_Pinning
 ---
 
-HTTP 公钥锁定（HPKP）是一种安全功能，它告诉 Web 客户端将特定加密公钥与某个 Web 服务器相关联，以降低使用伪造证书进行 MITM 攻击的风险。
+{{QuickLinksWithSubpages("/zh-CN/docs/Web/Security")}}
 
-为确保 TLS 会话中使用的服务器公钥的真实性，此公钥将包装到 X.509 证书中，该证书通常由证书颁发机构（CA）签名。诸如浏览器之类的 Web 客户端信任许多这些 CA，它们都可以为任意域名创建证书。如果攻击者能够攻击单个 CA，则他们可以对各种 TLS 连接执行 MITM 攻击。HPKP 可以通过告知客户端哪个公钥属于某个 Web 服务器来规避 HTTPS 协议的这种威胁。
+**证书透明度**（Certificate Transparency，CT）是一个开放的框架，旨在监测和防止证书的误发。它在 [RFC 9162](https://www.rfc-editor.org/rfc/rfc9162) 中定义。有了证书透明度机制，新颁发的证书会被“记录”到公开运行的、通常是独立的 *CT 日志*中——这些日志保持着仅添加的、有密码学保证的已颁发 TLS 证书记录。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/security/certificate_transparency/index.md:8]:
```suggestion
**证书透明度**（Certificate Transparency，CT）是一个开放的框架，旨在监测和防止证书的误发。它在 [RFC 9162](https://www.rfc-editor.org/rfc/rfc9162) 中定义。有了证书透明度机制，新颁发的证书会被“记录”到公开运行的、通常是独立的 *CT 日志*中——这些日志保持着仅允许添加、有密码学保证的已颁发 TLS 证书记录。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**